rpaas/manager.go

```diff
@@ -174,6 +176,32 @@ func (m *k8sRpaasManager) GetInstance(name string) (*v1alpha1.RpaasInstance, err
 	return &list.Items[0], nil
 }
 
+func (m *k8sRpaasManager) GetPlan(name string) (*v1alpha1.RpaasPlan, error) {
+	planList := &v1alpha1.RpaasPlanList{}
+	err := m.cli.List(m.ctx, client.MatchingField("metadata.name", name), planList)
```

Comment:
This logic of listing instead of using Get isn't needed here since RpaasPlan resources are cluster scoped: https://github.com/tsuru/rpaas-operator/blob/06d49b06246b2ebf9512203754829a1ee92e9636/deploy/crds/extensions_v1alpha1_rpaasplan_crd.yaml#L13
Simply using Get with an empty namespace will work here. This logic is only used for RpaasInstance because while they are namespaced we want to ensure the name does not repeat in other namespaces due to compatibility with tsuru integration.

Reply:
Ops... of course you're right. I was biased by `GetInstance` method 😃 Thanks!